selfservice/strategy/link/strategy_recovery_test.go

```diff
@@ -244,6 +245,50 @@ func TestRecovery(t *testing.T) {
 		})
 	})
 
+	t.Run("description=should require a valid email to be sent", func(t *testing.T) {
+		var check = func(t *testing.T, actual string, value string) {
+			assert.EqualValues(t, node.RecoveryLinkGroup, gjson.Get(actual, "active").String(), "%s", actual)
+			assert.EqualValues(t, fmt.Sprintf("%q is not valid \"email\"", value),
+				gjson.Get(actual, "ui.nodes.#(attributes.name==email).messages.0.text").String(),
+				"%s", actual)
+		}
+		var v1 = "abc"
```

Comment:
Maybe make this a loop?

```
for _, email := range []string{"\\", "asdf", "..."} {
 		var values = func(v url.Values) {
 			v.Set("email", email)
 		}

 		t.Run("type=browser", func(t *testing.T) {
 			check(t, expectValidationError(t, false, values), v1)
 		})

 		t.Run("type=api", func(t *testing.T) {
 			check(t, expectValidationError(t, true, values), v1)
 		})
}
```

Reply:
thanks for the comments. addressed those. The ci/circleci: cli  check keeps failing. Do you know how to fix? @aeneasr